blog/2015-05-26-port-monitoring.adoc

```diff
@@ -0,0 +1,58 @@
+
+image::/blog/images/20150521-monitoring-view.png[]
+
+Clicking on each individual request will show you both the request on the left side, and the response on the right. It will preview images that were requested, or simply display the contents of any text requests and responses.  Headers are also shown for both the request and the response. 
+
+=== Integration with the Servers View
+
+While everything seems to work nicely in the above simple example, tools need to make sure that they make requests to the new Local Port rather than the actual destination.  Our recent commits focussed on this deficiency. 
+
+To see this in action with the Servers View, you'll want to open your Servers View. You hopefully already have a WildFly instance installed there, which you can go ahead and start now. When you right-click on your Server Adapter, you'll see a "Monitoring" menu item. We currently have support for monitoring requests on two ports: the HTTP port (8080, typically) and the management port (9990). 
+
+image::/blog/images/20150521-monitoring-server.png[]
+
+After that, you can create a web project with a servlet or a simple html file and deploy it to the server. If you then right-click on your resource and select Run As -> Run On Server, you should see the browser open up to the URL http://localhost:7460/YourProject/index.html (or similar URL). In this way, the tools have made sure to make all requests to the Local Port when port monitoring is enabled, but to make all requests to the actual destination port when monitoring is disabled. 
+
+When your server is in a management-enabled profile, requests made to the WildFly Management port will also show up in the TCP/IP monitor for inspection. 
+
+== Conclusion
+
+So there we have it: A long-standing omission has finally been put in place. What's more, other server adapters don't integrate with this feature as completely as we do. If you were to run the same experiment on the Tomcat adapter, and execute the "Run As -> Run on Server" action for a web project's servlet or html file, the browser will still open up on port 8080, rather than the Local Port which can be used for monitoring. When targeting JBoss and WildFly servers, though, these actions behave as a user would expect and hope. 
+
+So get in there and monitor some traffic!  Grab a nightly build from our link:http://tools.jboss.org/downloads/jbosstools/mars/4.3.0.Nightly.html[downloads] section, or just use our nightly update site directly:
+  
+     http://download.jboss.org/jbosstools/updates/nightly/mars/  
+
+Related JIRA: https://issues.jboss.org/browse/JBIDE-19755[JBIDE-19755 - Implement ServerPort portion of wst.servertools API]
+
+ Rob Stryker
```

Comment:
this creates verbatim text ;)

anyways I fixed that and merged it in and it should be live anytime soon.